ReText/__init__.py

```diff
@@ -92,6 +92,7 @@ def getBundledIcon(iconName):
 	'styleSheet': '',
 	'syncScroll': True,
 	'tabBarAutoHide': False,
+	'titleBarFullPath': False,
```

Comment:
Can you please keep this file sorted? I.e. the new option should go after `tabWidth`.

Also please document this new option in `configuration.md`.

Reply:
I'll try to do better in future